pkg/innerring/processors/common/node.go

```diff
@@ -0,0 +1,9 @@
+package common
+
+// NodeStatus represents status of the local Inner Ring node in the NeoFS network.
+type NodeStatus interface {
```

Comment:
Yeah, otherwise we will copy this in almost all processors. The only alternative I see is to eject Alphabet-only check from processors and perform it preliminaliry on event "distillation".